api/src/main/java/org/eclipse/microprofile/config/Config.java

```diff
@@ -181,6 +181,50 @@
         return getOptionalValue(propertyName, arrayType).map(Arrays::asList);
     }
 
+    /**
+     * Return the resolved configuration properties type with the specified prefix. 
+     * The type declaration can be annotated with 
+     * {@linkplain org.eclipse.microprofile.config.inject.ConfigProperties @ConfigProperties}. 
+     * If the type is annotated with {@linkplain org.eclipse.microprofile.config.inject.ConfigProperties @ConfigProperties}, 
```

Comment:
I think we can simplify this a little bit, just:

"The prefix method parameter overrides the any other prefix set on the type".

Reply:
done